presto-main/src/main/java/com/facebook/presto/operator/aggregation/LongDoubleConverterUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation;
+
+class LongDoubleConverterUtil
+{
+    private static final long SIGNED_LONG_MAX = 0x7fffffffffffffffL;
+
+    private LongDoubleConverterUtil() {}
+
+    /**
+     * Returns the double value corresponding to the (unsigned) bit representation of a long
+     */
+    public static double longToDouble(long value)
```

Comment:
I looked at how some libraries name this function (e.g. Lucene). Let's call it `sortableLongtoDouble`.